.circleci/config.yml

```diff
@@ -18,3 +32,154 @@ jobs:
           path: cypress/screenshots
       - store_artifacts:
           path: cypress/videos
+  test_gatsby:
+    docker:
+      - image: cypress/browsers:node12.13.0-chrome80-ff74
```

Comment:
I used the same image as we use for the integration tests. I would argue this is not needed, but I got permission error on some others I tried (node alpine, ubuntu, ...), so I figured sticking to the one we are already using wouldn't hurt. What do u think?

Reply:
Good call, I will have a look!